Control.cpp

```diff
@@ -25,7 +30,7 @@ double Control::getDesiredAngle(double desire) {
   }
 
```

Comment:
You don't need the space between the if and the else.
if {
}
else {
}

not 
if {
}

else {
}